docassemble/tclpgoogledocsmerger/data/questions/search_and_download.yml

```diff
@@ -78,6 +82,41 @@ code: |
     if help_link:
       selectable_fields[-1]['help'] = f'[More info here]({help_link})'
 ---
+code: |
+  # Take the interesting fields and make them 2 dimensional so easier to view in XLSX
+  stuff_to_snapshot = {
+     'start_time': str(start_time().format('yyyy-MM-dd')),
+     }    
+  # Get location
+  try:
+    import requests
+    response = requests.get(f"https://geolocation-db.com/json/{device(ip=True)}&position=true").json()
+    stuff_to_snapshot['country'] = response.get('country_code')
+    stuff_to_snapshot['state'] = response.get('state')
+    stuff_to_snapshot['city'] = response.get('city')
+    stuff_to_snapshot['latitude'] = response.get('latitude')
+    stuff_to_snapshot['longitude'] = response.get('longitude')
+  except:
+    stuff_to_snapshot['country'] = 'UNKNOWN'
```

Comment:
Should probably save the IP if we can't get the location information, if that's allowed?